manifests/config.pp

```diff
@@ -3,34 +3,41 @@
 # @api private
 class chrony::config (
   $bindcmdaddress       = $chrony::bindcmdaddress,
-  $cmdport              = $chrony::cmdport,
   $cmdacl               = $chrony::cmdacl,
+  $cmdport              = $chrony::cmdport,
   $commandkey           = $chrony::commandkey,
   $config               = $chrony::config,
   $config_template      = $chrony::config_template,
   $config_keys          = $chrony::config_keys,
   $config_keys_template = $chrony::config_keys_template,
+  $chrony_password      = $chrony::chrony_password,
   $config_keys_owner    = $chrony::config_keys_owner,
   $config_keys_group    = $chrony::config_keys_group,
   $config_keys_mode     = $chrony::config_keys_mode,
   $config_keys_manage   = $chrony::config_keys_manage,
-  $chrony_password      = $chrony::chrony_password,
   $keys                 = $chrony::keys,
+  $local_stratum        = $chrony::local_stratum,
   $log_options          = $chrony::log_options,
   $refclocks            = $chrony::refclocks,
-  $mailonchange         = $chrony::mailonchange,
-  $threshold            = $chrony::threshold,
-  $lock_all             = $chrony::lock_all,
   $peers                = $chrony::peers,
   $servers              = $chrony::servers,
   $pools                = $chrony::pools,
+  $makestep_seconds     = $chrony::makestep_seconds,
```

Comment:
This looks like an unrelated change.

Reply:
This parameter is required by the templates.